### ISSUE TITLE
Media.ImportStream() overload with filename cannot be used for Cloud/SaaS, which is very inconvenient

### ISSUE THREAD

**Issue by dnpb** [microsoft/AL#6890]
Similar to https://github.com/microsoft/AL/issues/6568
which was discussed on https://www.yammer.com/dynamicsnavdev/#/Threads/show?threadId=1102694460448768

I previously opened this Yammer thread too: https://www.yammer.com/dynamicsnavdev/#/Threads/show?threadId=1560464358563840

`Media.ImportStream(Stream: InStream, Description: Text, MimeType: Text, FileName: Text)` is only available OnPrem, not for Cloud/SaaS dev. Must this be so?

It doesn't make sense to me! The import will usually originate as a file, in my case from `UploadIntoStream` and its `FromFile` argument. Thus, we have a file name to give it, and will want to use said filename again when exporting... Meaning either we have to stash it in some other field (more work) or manually set/retrieve the Filename from the `Tenant Media` table. I'm now doing the latter, but it is boilerplate and looks wrong, as feel that's a table we shouldn't really have to poke at directly, right?
  
Please can you restore this overload with `FileName` for SaaS development, so we too can have the imported filename available for archival/convenience/whatever, as it was done with #6568? Thanks :-)

**Comment by thloke** [comment]
Hi, we've discussed this more internally. We could make it available, but it's not as straightforward as flipping the availability from OnPrem to cloud since we need to do a security assessment of the impact of the change. I agree that at a glance it should be harmless, but we still need to do a full assessment with the team that owns the runtime behaviour of this method. Since this is an (admittedly useful) enhancement rather than a bug, we decided to not accept this through the Github channel.

What this means is that the best way to get traction on this from all the correct teams is either through BCIdeas (probably a larger general request that bundles other related asks around any other methods that would be nice to move from OnPrem to Cloud would be best so we can do all of them at once) or Collaborate. 

I know this is probably not the response you were hoping for, so I apologize in advance. ):
